src/test/java/TurnUseCases/TradeUseCase/TradeOfferTest.java

```diff
@@ -0,0 +1,156 @@
+package TurnUseCases.TradeUseCase;
+
+import GameEntities.Player;
+import GameEntities.Tiles.ColorPropertyTile;
+import GameEntities.Tiles.Property;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.*;
+
+public class TradeOfferTest {
+    Player player1 = new Player("player1", "player1", 1500);
+
+    ColorPropertyTile fakeStreet = new ColorPropertyTile("red", "fakeStreet",
```

Comment:
`ColorPropertyTile has been updated to use an array for rent instead of different parameters as of e40f2aae15067f77e9d9bc59dfa160ec495e37f5